modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +62,18 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const copyAppropriateAttributes = <T extends HTMLElement> (original: SugarElement<T>, clone: SugarElement<T>): void => {
+  Obj.each(transferableAttributes, (_keyArray: string[], attributeName: string) =>
+    Attribute.getOpt(original, attributeName)
+      .filter((attribute) =>
```

Comment:
This is not our normal code style. This is a short check so it should be on a single line.